File: Java&Kotlin lecture/src/test/java/com/java/StudentTest.java
package com.java;

import com.java.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void getHighestTest() {
        Student student = new Student("", "Peter", 89, 72);
        Assertions.assertEquals(89, student.getHighest());
    }

    @Test
    public void getAverageTest() {
        Student student = new Student("", "Peter", 89, 72);
        Assertions.assertEquals((89 + 72) / 2, student.getAverage());
    }
}

File: Java&Kotlin lecture/src/test/kotlin/com/kotlin/StudentTest.kt
package com.kotlin

import com.kotlin.student.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun getHighestTest() {
        val student = Student("", "Peter", 89, 72)
        Assertions.assertEquals(89, student.getHighest())
    }

    @Test
    fun getAverageTest() {
        val student = Student("", "Peter", 89, 72)
        Assertions.assertEquals((89 + 72) / 2, student.getAverage())
    }
}